lib/v2/screens/dashboard/wallet_screen.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/bloc.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/rates_bloc.dart';
+import 'package:seeds/v2/components/profile_avatar.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/i18n/explore_screens/explore/explore.i18n.dart';
+import 'package:seeds/v2/navigation/navigation_service.dart';
+import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/wallet_bloc.dart';
+import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/wallet_event.dart';
+import 'package:seeds/v2/screens/dashboard/wallet_header.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+
+// Widgets to be moved to v2
+import 'package:seeds/widgets/v2_widgets/dashboard_widgets/receive_button.dart';
+import 'package:seeds/widgets/v2_widgets/dashboard_widgets/send_button.dart';
+
+enum TransactionType { income, outcome }
+
+/// Wallet SCREEN
+class WalletScreen extends StatelessWidget {
+  const WalletScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (_) => WalletBloc()..add(const RefreshDataEvent()),
+      child: RefreshIndicator(
+          child: Scaffold(
+            appBar: buildAppBar(context) as PreferredSizeWidget?,
+            body: ListView(
+              // TODO(n13): Use exact measurements from figma
+              children: <Widget>[
+                const SizedBox(height: 15),
+                const WalletHeader(),
+                const SizedBox(height: 20),
+                buildSendReceiveButton(context),
+                const SizedBox(height: 20),
+                walletBottom(context),
```

Comment:
@gguij004  I guess `buildAppBar`, `buildSendReceiveButton` and `walletBottom` functions will be converted to Widget classes and moved to components.

Reply:
yes